src/PrestaShopBundle/Form/Admin/Configure/AdvancedParameters/Employee/EmployeeType.php

```diff
@@ -166,23 +212,43 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                     SwitchType::class,
                     [
                         'required' => false,
+                        'label' => $this->trans('Active', 'Admin.Global'),
+                        'help' => $this->trans('Allow or disallow this employee to log in to the Admin panel.', 'Admin.Advparameters.Help'),
                     ]
                 )
                 ->add(
                     'profile',
                     ChoiceType::class,
                     [
                         'choices' => $this->profilesChoices,
+                        'label' => $this->trans('Permission profile', 'Admin.Advparameters.Feature'),
+                        'attr' => [
+                            'data-admin-profile' => $this->superAdminProfileId,
+                            'data-get-tabs-url' => $this->router->generate('admin_employees_get_tabs'),
+                        ],
                     ]
                 )
             ;
-
             if ($this->isMultistoreFeatureActive) {
                 $builder->add('shop_association', ShopChoiceTreeType::class, [
+                    'label' => $this->trans('Shop association', 'Admin.Global'),
+                    'help' => $this->trans('Select the shops the employee is allowed to access.', 'Admin.Advparameters.Help'),
                     'required' => false,
                 ]);
             }
         }
+
+        $builder
+            ->add('default_page', ChoiceType::class, [
+                'choices' => $this->tabChoices,
+                'label' => $this->trans('Default page', 'Admin.Advparameters.Feature'),
+                'help' => $this->trans('This page will be displayed just after login.', 'Admin.Advparameters.Help'),
+                'attr' => [
+                    'data-minimumResultsForSearch' => '7',
```

Comment:
```suggestion
                    'data-minimum-results-for-search' => '7',
```
But yes, it requires a const :thinking:

Reply:
If I search for minimumResultsForSearch, I have a lot results with camelCase. I'm wondering if it's not because jquery.select requires it like that